Packaging/Packaging.proj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="CreateStepPackage" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This project and the solution file both ref version 14 which hasn't shipped yet. Will the solution and project still be usable in VS2013?